prometheuspvesd/test/fixtures/fixtures.py

```diff
@@ -222,12 +240,40 @@ def qemus():
     ]
 
 
+@pytest.fixture
+def instance_config():
+    return {
+        "name":
+            "102.example.com",
+        "description":
+            '{"groups": "test-group"}',
+        "net0":
+            "virtio=D8-85-75-47-2E-8D,bridge=vmbr122,ip=192.0.2.25,ip=2001:db8::6666:7777:8888",
+        "cpu":
+            2,
+        "cores":
+            2
+    }
```

Comment:
```suggestion
    return {
        "name": "102.example.com",
        "description": '{"groups": "test-group"}',
        "net0": "virtio=D8-85-75-47-2E-8D,bridge=vmbr122,ip=192.0.2.25,ip=2001:db8::6666:7777:8888",  # noqa
        "cores": 2
    }
```

Reply:
I can do that, but then flake 8 isn´t happy with the net0 line as it is too long :/
But I can shorten the ipv6 address a bit, and keep it valid

Reply:
Thats whats the `# noqa´ is for :)